src/index.js

```diff
@@ -44,7 +49,7 @@ export default class SaucelabsConnector {
         return JSON.parse(response.body).concurrency[this.username].remaining.overall;
     }
 
-    async startBrowser (browser, url, jobName, timeout = null) {
+    async startBrowser (browser, url, timeout = null) {

```

Comment:
We should can an ability to run tasks with different jobNames (builds or tags) with the same saucelabsConnector instance. So, this args relates to the `startBrowser` fn. Let's move them from the constructor to this function.
They can be looked like this:
```js
async startBrowser (browser, url, timeout = null, { jobName, build, tags })
```
And they should be optional